### PR TITLE
#24 Created a pull request template

### LINKED ISSUE #24: Add Pull Request Template
**User Story**
As a developer
I want to have a template for pull requests
So when I make a pull request I include all the relevant information and it is easy for others to understand.

**Priority**
[x]: High
[]: Medium
[]: Low

**Describe the solution you want to implement**
Create a PR template file to put in the GitHub repo.

**Describe alternatives you've considered**
None

**Issue Dependencies**
This should be done before any code issues so PR's look good.

**Additional context**
Have a look at this for help: https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

### PR DESCRIPTION
**Pull request by margeobur**
Yo dawg, I heard you like pull requests, so we made a pull request for your pull request template so you can make more pull requests.

Closes #24 - Add Pull Request Template 

This PR introduces a simple template for future PRs. It includes:
* A basic layout for PRs including
  * The related issue
  * A description of the introduced changes
* Suggestions on what to include

To be honest I'm not sure if this is even close to what's appropriate - that's why I'm making a PR, so you guys can give feedback

### CODE REVIEW

**Comment by SeanSpires** [comment]
A simple template, but I think simple is good for PRs at this stage.

**Review by SeanSpires — APPROVED** [review]

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Closes (issue name)#(number).
```
Comment (williamlixu) [inline docs/pull_request_template.md]: Unfortunately this doesn't automatically close the issue, it has to be just "Closes #<number>". GitHub does show you the issue when you hover over the number tho which is nice enough :)

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Closes #(number) - (issue name)
```
Comment (williamlixu) [inline docs/pull_request_template.md]: Bit of a hassle to get the issue name and GitHub will actually let you hover over to see the issue anyway

File: docs/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+This PR introduces... (write a simple description of the added feature or solution to the bug)
+- you can have a bullet list with distinct bits of functionality
+- a good thing to mention here would be API calls made or DB tables accessed
+
+If necessary, describe what any new files are for
+
+@mention your team
```
Comment (williamlixu) [inline docs/pull_request_template.md]: Probably not important to mention your team, you should request them if they are needed to do the code review.

File: docs/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Closes #(number) - (issue name)
+
+This PR introduces... (write a simple description of the added feature or solution to the bug)
```
Comment (williamlixu) [inline docs/pull_request_template.md]: Could have headings like the issue templates and might make it easier to quickly glance over a PR

**Review by williamlixu — CHANGES_REQUESTED** [review]
Looks good, just some small fixes

File: docs/pull_request_template.md (hunk 2 above)
Comment (williamlixu) [reply docs/pull_request_template.md]: Also can this file be moved to `/.github` folder to go with the other templates

File: docs/pull_request_template.md (hunk 3 above)
Comment (williamlixu) [reply docs/pull_request_template.md]: Could also add something about testing like **Describe any testing done** as a heading

**Review by williamlixu — APPROVED** [review]
LGTM!

**Pull request merged**